.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         touch -t ${touchtime} "${filename}"
         ls -la --time-style=full-iso "${filename}"
     - name: Generate
-      run: make generate
+      run: make generate genjsonschema
```

Comment:
Why not adding it to `generate` target in Makefile?

Reply:
I hesitated, and I can do it.
I went with not doing it because the command takes a bit of time to run, and needs to be executed much less often than generate.

Reply:
Reasonable to me if it takes more than a few seconds.
Otherwise I would add it to `generate` target.

Reply:
The command runs the linter. If we remove it, running it on generate seems fine

Reply:
Makes sense. I think not running linter will also be clenaer.